lib/rules/at-rule-descriptor-value-no-invalid/index.mjs

```diff
@@ -0,0 +1,64 @@
+import { lexer } from 'css-tree';
+
+import { declarationValueIndex } from '../../utils/nodeFieldIndices.mjs';
+import { nestingSupportedAtKeywords } from '../../reference/atKeywords.mjs';
+import report from '../../utils/report.mjs';
+import ruleMessages from '../../utils/ruleMessages.mjs';
+import validateOptions from '../../utils/validateOptions.mjs';
+
+const UNSUPPORTED_NESTING_REGEX = new RegExp(
+	`^((?!${[...nestingSupportedAtKeywords.values()].join('|')}).)*$`,
```

Comment:
question

Are you sure you want `.` and `*`?
Wouldn't something like `^(?:${[...nestingSupportedAtKeywords.values()].join('|')})+$` be more accurate?

Reply:
Wouldn't your example end up checking only `nestingSupportedAtKeywords`? 
That pattern would cause the tests to fail.